### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by grantwforsythe**
### Description

Fix a typo and clarify an example.

### Motivation

Improves readability for the reader.

### CODE REVIEW

File: files/en-us/web/guide/ajax/getting_started/index.md [hunk 1]
```diff
@@ -31,9 +31,12 @@ To make an [HTTP](/en-US/docs/Web/HTTP) request to the server with JavaScript, y
...
 ```
 
-After making a request, you will receive a response back. At this stage, you need to tell the XMLHttp request object which JavaScript function will handle the response, by setting the `onreadystatechange` property of the object and naming it after the function to call when the request changes state, like this:
+After making a request, you will receive a response back. At this stage, you need to tell the XMLHttpRequest object which JavaScript function will handle the response, by setting the `onreadystatechange` property of the object and naming it after the function to call when the request changes state, like this:
 
 ```js
+function nameOfTheFunction() {
+    // Process the server response here.
+}
 httpRequest.onreadystatechange = nameOfTheFunction;
```
Comment (Josh-Cena) [inline files/en-us/web/guide/ajax/getting_started/index.md:40]: If we provide the actual function stub, the `nameOfTheFunction` placeholder doesn't make sense anymore.

```suggestion
httpRequest.onreadystatechange = () => {
  // Process the server response here.
};
```

File: files/en-us/web/guide/ajax/getting_started/index.md [hunk 2]
```diff
@@ -31,9 +31,12 @@ To make an [HTTP](/en-US/docs/Web/HTTP) request to the server with JavaScript, y
 const httpRequest = new XMLHttpRequest();
 ```
 
-After making a request, you will receive a response back. At this stage, you need to tell the XMLHttp request object which JavaScript function will handle the response, by setting the `onreadystatechange` property of the object and naming it after the function to call when the request changes state, like this:
+After making a request, you will receive a response back. At this stage, you need to tell the XMLHttpRequest object which JavaScript function will handle the response, by setting the `onreadystatechange` property of the object and naming it after the function to call when the request changes state, like this:
```
Comment (Josh-Cena) [inline files/en-us/web/guide/ajax/getting_started/index.md:34]:
```suggestion
After making a request, you will receive a response back. At this stage, you need to tell the `XMLHttpRequest` object which JavaScript function will handle the response, by setting the `onreadystatechange` property of the object and naming it after the function to call when the request changes state, like this:
```

File: files/en-us/web/guide/ajax/getting_started/index.md [hunk 3]
```diff
@@ -31,10 +31,12 @@ To make an [HTTP](/en-US/docs/Web/HTTP) request to the server with JavaScript, y
 const httpRequest = new XMLHttpRequest();
 ```
 
-After making a request, you will receive a response back. At this stage, you need to tell the XMLHttp request object which JavaScript function will handle the response, by setting the `onreadystatechange` property of the object and naming it after the function to call when the request changes state, like this:
+After making a request, you will receive a response back. At this stage, you need to tell the `XMLHttpRequest` object which JavaScript function will handle the response, by setting the `onreadystatechange` property of the object and naming it after the function to call when the request changes state, like this:
```
Comment (Josh-Cena) [inline files/en-us/web/guide/ajax/getting_started/index.md:34]:
```suggestion
After making a request, you will receive a response back. At this stage, you need to tell the `XMLHttpRequest` object which JavaScript function will handle the response, by setting the `onreadystatechange` property of the object to the function called when the request changes state, like this:
```

File: files/en-us/web/guide/ajax/getting_started/index.md [hunk 4]
```diff
@@ -31,10 +31,12 @@ To make an [HTTP](/en-US/docs/Web/HTTP) request to the server with JavaScript, y
...
 ```
 
-After making a request, you will receive a response back. At this stage, you need to tell the XMLHttp request object which JavaScript function will handle the response, by setting the `onreadystatechange` property of the object and naming it after the function to call when the request changes state, like this:
+After making a request, you will receive a response back. At this stage, you need to tell the `XMLHttpRequest` object which JavaScript function will handle the response, by setting the `onreadystatechange` property of the object to the function called when the request changes state, like this:
 
 ```js
-httpRequest.onreadystatechange = nameOfTheFunction;
+httpRequest.onreadystatechange = () => {
+  // Process the server response here.
+};
```
Comment (Josh-Cena) [inline files/en-us/web/guide/ajax/getting_started/index.md:39]:
```suggestion
function handler() {
  // Process the server response here.
}

httpRequest.onreadystatechange = handler;
```

**Review by Josh-Cena — APPROVED** [review]
I made a mistake in my previous review because I didn't read the next section which talks about using anonymous functions. I've done the changes myself. Thank you, and congratulations on landing your first PR here—welcome aboard!

**Comment by grantwforsythe** [comment]
@Josh-Cena Thanks for the guidance!

**Pull request merged**